classes/ActionScheduler_LogEntry.php

```diff
@@ -4,12 +4,31 @@
  * Class ActionScheduler_LogEntry
  */
 class ActionScheduler_LogEntry {
-	protected $action_id = '';
-	protected $message = '';
+	protected $action_id =  '';
+	protected $message =  '';
+	protected $date;
 
-	public function __construct( $action_id, $message ) {
+	/**
+	 * Constructor
+	 *
+	 * @param mixed  $action_id	Action ID
+	 * @param string $message   Message
+	 * @param Datetime $date    Datetime object with the time when this log entry was created. If this parameter is
+	 *                          not provided a new Datetime object (with current time) will be created.
+	 */
+	public function __construct( $action_id, $message, Datetime $date = null ) {
 		$this->action_id = $action_id;
-		$this->message = $message;
+		$this->message   = $message;
+		$this->date      = $date ? $date : new Datetime;
+	}
+
+	/**
+	 * Returns the date when this log entry was created
+	 *
+	 * @return Datetime
+	 */
+	public function get_date() {
+		return $this->date;
 	}
 
 	public function get_action_id() {
```

Comment:
Let's go ahead and add docblocks to the other methods here.

Reply:
I've opened https://github.com/Prospress/action-scheduler/issues/125 to address this.

Code review isn't the time to raise or address issues in code that is not being added/modified by the PR.

Reply:
I'm completely fine with doing docblocks separately. I've mainly mentioned them here just to make sure everything is covered. Personally, I find it easier to include simple things like docblocks when I'm already working in a file, but I can understand that's not the same for everyone.